z3/include/z3_solver.h

```diff
@@ -1,23 +1,39 @@
-#ifndef SMT_CVC4_SOLVER_H
```

Comment:
It's probably a good idea to have header guards. I checked and don't see them for Yices or MathSat, so maybe after this we can add them for those, too.

Reply:
Good catch, I have a preference for `#pragma once` these days. Thoughts?

Reply:
Ah, I forgot that there were `#pragma once`'s in the header files. I don't really have a preference.

Reply:
Oh there's probably some missing for sure so it's still worth checking, but between the two I think `pragma`s are cleaner.